cristin-import/src/main/java/no/unit/nva/cristin/mapper/artisticproduction/ArtisticProductionType.java

```diff
@@ -20,7 +20,12 @@
 @Getter
 @Setter
 @AllArgsConstructor(access = AccessLevel.PACKAGE)
-@JsonIgnoreProperties({"produkttypenavn", "produkttypenavn_engelsk", "produkttypenavn_nynorsk"})
+@JsonIgnoreProperties({"produkttypenavn",
```

Comment:
Fikser bug:

java.lang.IllegalArgumentException: Unrecognized field "produksjonstypenavn" (class no.unit.nva.cristin.mapper.artisticproduction.ArtisticProductionType), not marked as ignorable (one known property: "produkttypekode"]) at [Source: UNKNOWN; byte offset: #UNKNOWN] (through reference chain: no.unit.nva.cristin.mapper.CristinObject["type_kunstneriskproduksjon"]->no.unit.nva.cristin.mapper.artisticproduction.CristinArtisticProduction["produksjonstype"]->no.unit.nva.cristin.mapper.artisticproduction.ArtisticProductionType["produksjonstypenavn"]) at com.fasterxml.jackson.databind.ObjectMapper._convert(ObjectMapper.java:4544) at com.fasterxml.jackson.databind.ObjectMapper.convertValue(ObjectMapper.java:4475) at no.unit.nva.cristin.mapper.CristinObject.lambda$fromJson$0(CristinObject.java:139) at nva.commons.core.attempt.Try.attempt(Try.java:33) at no.unit.nva.cristin.mapper.CristinObject.fromJson(CristinObject.java:139) at nva.commons.core.attempt.Success.lambda$map$0(Success.java:43) at nva.commons.core.attempt.Try.attempt(Try.java:33) at nva.commons.core.attempt.Success.map(Success.java:43) at no.unit.nva.cristin.lambda.CristinEntryEventConsumer.jsonNodeToCristinObject(CristinEntryEventConsumer.java:254) at no.unit.nva.cristin.lambda.CristinEntryEventConsumer.parseCristinObject(CristinEntryEventConsumer.java:241) at no.unit.nva.cristin.lambda.CristinEntryEventConsumer.getCristinObject(CristinEntryEventConsumer.java:154) at no.unit.nva.cristin.lambda.CristinEntryEventConsumer.lambda$processEvent$1(CristinEntryEventConsumer.java:145) at nva.commons.core.attempt.Try.attempt(Try.java:33) at no.unit.nva.cristin.lambda.CristinEntryEventConsumer.processEvent(CristinEntryEventConsumer.java:145) at nva.commons.core.attempt.Success.lambda$map$0(Success.java:43) at nva.commons.core.attempt.Try.attempt(Try.java:33) at nva.commons.core.attempt.Success.map(Success.java:43) at no.unit.nva.cristin.lambda.CristinEntryEventConsumer.processMessage(CristinEntryEventConsumer.java:139) at java.base/java.util.stream.ReferencePipeline$3$1.accept(Unknown Source) at java.base/java.util.ArrayList$ArrayListSpliterator.forEachRemaining(Unknown Source) at java.base/java.util.stream.AbstractPipeline.copyInto(Unknown Source) at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(Unknown Source) at java.base/java.util.stream.ReduceOps$ReduceOp.evaluateSequential(Unknown Source) at java.base/java.util.stream.AbstractPipeline.evaluate(Unknown Source) at java.base/java.util.stream.ReferencePipeline.collect(Unknown Source) at no.unit.nva.cristin.lambda.CristinEntryEventConsumer.handleRequest(CristinEntryEventConsumer.java:101) at jdk.internal.reflect.GeneratedMethodAccessor45.invoke(Unknown Source) at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) at java.base/java.lang.reflect.Method.invoke(Unknown Source) at com.amazonaws.services.lambda.runtime.api.client.EventHandlerLoader$PojoMethodRequestHandler.handleRequest(EventHandlerLoader.java:285) at com.amazonaws.services.lambda.runtime.api.client.EventHandlerLoader$PojoHandlerAsStreamHandler.handleRequest(EventHandlerLoader.java:202) at com.amazonaws.services.lambda.runtime.api.client.EventHandlerLoader$2.call(EventHandlerLoader.java:905) at com.amazonaws.services.lambda.runtime.api.client.AWSLambda.startRuntime(AWSLambda.java:245) at com.amazonaws.services.lambda.runtime.api.client.AWSLambda.startRuntime(AWSLambda.java:197) at com.amazonaws.services.lambda.runtime.api.client.AWSLambda.main(AWSLambda.java:187) Caused by: com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException: Unrecognized field "produksjonstypenavn" (class no.unit.nva.cristin.mapper.artisticproduction.ArtisticProductionType), not marked as ignorable (one known property: "produkttypekode"]) at [Source: UNKNOWN; byte offset: #UNKNOWN] (through reference chain: no.unit.nva.cristin.mapper.CristinObject["type_kunstneriskproduksjon"]->no.unit.nva.cristin.mapper.artisticproduction.CristinArtisticProduction["produksjonstype"]->no.unit.nva.cristin.mapper.artisticproduction.ArtisticProductionType["produksjonstypenavn"]) at com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException.from(UnrecognizedPropertyException.java:61) at com.fasterxml.jackson.databind.DeserializationContext.handleUnknownProperty(DeserializationContext.java:1138) at com.fasterxml.jackson.databind.deser.std.StdDeserializer.handleUnknownProperty(StdDeserializer.java:2224) at com.fasterxml.jackson.databind.deser.BeanDeserializerBase.handleUnknownProperty(BeanDeserializerBase.java:1709) at com.fasterxml.jackson.databind.deser.BeanDeserializerBase.handleUnknownVanilla(BeanDeserializerBase.java:1687) at com.fasterxml.jackson.databind.deser.BeanDeserializer.vanillaDeserialize(BeanDeserializer.java:320) at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserialize(BeanDeserializer.java:177) at com.fasterxml.jackson.databind.deser.impl.MethodProperty.deserializeAndSet(MethodProperty.java:129) at com.fasterxml.jackson.databind.deser.BeanDeserializer.vanillaDeserialize(BeanDeserializer.java:314) at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserialize(BeanDeserializer.java:177) at com.fasterxml.jackson.databind.deser.impl.MethodProperty.deserializeAndSet(MethodProperty.java:129) at com.fasterxml.jackson.databind.deser.BeanDeserializer.vanillaDeserialize(BeanDeserializer.java:314) at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserialize(BeanDeserializer.java:177) at com.fasterxml.jackson.databind.ObjectMapper._convert(ObjectMapper.java:4539) ... 34 more